### DISCUSSION TITLE
How to keep Connector Connection on Page refresh

### DISCUSSION THREAD

**Discussion by almatutor** [wevm/wagmi#3732] [Q&A] [answered]
I want to keep connector connection on page refresh. 
I want to invoke signTypedDataAsync function to sign and send data to backend, but when I refresh page, because ConnectorConnection is closed, I have to connect again.
Is there anyone who can help me?

**Comment by cryptohero92** [comment, marked as answer]
Please try [this](https://web3auth.io/community/t/keep-web3auth-session-persistent-after-page-refresh/3262/3) to keep auto connection.

**Reply by almatutor** [reply]
okay, I will check it.
